modin/experimental/core/execution/native/implementations/hdk_on_native/dataframe/utils.py

```diff
@@ -379,7 +381,74 @@ def to_empty_pandas_df(df):
     return index_cols, exprs, new_dtypes, merged.columns
 
 
-def arrow_to_pandas(at: pa.Table) -> pd.DataFrame:
+def mangle_index_names(names: List[ColNameCodec._COL_NAME_TYPE]) -> List[str]:
```

Comment:
why was it moved out of the dataframe class? I would understand if you moved both mangling and [de-mangling](https://github.com/modin-project/modin/blob/d35e61f48ae89c9fb7716564bb3e284d14cac8e1/modin/experimental/core/execution/native/implementations/hdk_on_native/dataframe/dataframe.py#L2598) functions, but why only one?

Reply:
if we choose to move these out of the dataframe class, maybe we should make them methods of the `ColNameCodec` class?

Reply:
It has been moved because it's required for the new function concat_index_names().
Moved to  the ColNameCodec class.